providers/openai.js

```diff
@@ -63,19 +63,43 @@ class OpenAi extends Provider {
 
         `);
 			setTimeout(() => {
-				this.getWebview().executeJavaScript(`
-          // Get the root element
-          const root = document.querySelector(':root');
+        // Get the current dark mode setting from the store
+        const isDarkMode = store.get('darkMode', false);
 
-          // Set the color-scheme CSS variable
-          root.style.setProperty('--color-scheme', 'dark');
+        // Toggle dark mode based on the current setting
+        this.toggleDarkMode(isDarkMode);
+      }, 300);
+		});
+	}
 
-          // Add the .dark class to the body
-          document.body.classList.add('dark');
+  static toggleDarkMode(isDarkMode) {
```

Comment:
This won't work for Claude, Bing, and a few others that don't support the system dark mode color property.

But for those providers we can just move the custom dark mode CSS we'd put together into toggleDarkMode at their level. I can take a stab at that now while I look at the shortcut thing.

Reply:
Makes sense. Thank you for the help!